src/module/item/kit/sheet.ts

```diff
@@ -17,19 +16,15 @@ export class KitSheetPF2e extends ItemSheetPF2e<KitPF2e> {
     }
 
     override async getData(options?: Partial<DocumentSheetOptions>) {
-        const sheetData = await super.getData(options);
-        const traits = createSheetTags(CONFIG.PF2E.classTraits, sheetData.data.traits);
-
         return {
-            ...sheetData,
+            ...(await super.getData(options)),
```

Comment:
:open_mouth:

Reply:
That should be correct, but I looked again and I forgot to remove the unnecessary template paths from there, so uh, updated!